README.md

```diff
@@ -68,6 +69,23 @@ $ sudo find / -name qmake
 ## Run:
 ``` ./start.sh [options] [script file name] ```
 
+## Documentation generation
+### Download and install [ldoc](stevedonovan.github.io/ldoc/manual/doc.md.html)
+```
+sudo apt install luarocks
+sudo luarocks install luasec
+sudo luarocks install penlight
+sudo luarocks install ldoc
```

Comment:
@LuxoftAKutsan,
It's all lua librarires and the haver a runtime dependency,
So, installation order is not important.
Note: Formally the order right at this moment: ldoc requires penlight and ldoc uses (due to our config file) discount